task/minio-cli/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- chmouel
+- vdemeester
+reviewers:
+- chmouel
+- vdemeester
```

Comment:
not a blocker just a query :sweat_smile:, don't you want to put your name in the list of approvers or reviewers for this task since you are authoring?

Reply:
hahaha it makes sense :) I will add this then.